examples/tutorial/00_basic.py

```diff
@@ -14,11 +14,12 @@
 
 # Initialize a connection to the Scanner database. Loads configuration from the
 # ~/.scanner.toml configuration file.
-with Database() as db:
+with Database(master="crissy:5001", workers=["crissy:5002"], debug=True) as db:
```

Comment:
Probably shouldn't have this hardcoded in the tutorial.

Reply:
Yup.